### ISSUE TITLE
Adding Rocky Linux

### ISSUE THREAD

**Issue by tuxpeople** [giovtorres/kvm-install-vm#71]
Rocky Linux is missing.

I tried to add it myself, to make a pull request. As far as I can tell, it may work, but it's not in the osinfo-db:

```
ERR: Unknown OS variant 'rocky84'. Please update your osinfo-db.  See https://libosinfo.org/download for more information.
```

Question: How to proceed? Add an exception?

**Comment by tuxpeople** [comment]
If you could help me with that, I would make a PR. And also add AlmaLinux.

**Comment by ronzle** [comment]
i added Rocky8.5 and created a pull request
lets wait for the merge

**Comment by ronzle** [comment]
#74

**Comment by giovtorres** [comment]
Closed in #74
